qpc/scan/edit.py

```diff
@@ -58,14 +59,29 @@ def __init__(self, subparsers):
                                  metavar='DISABLE_OPTIONAL_PRODUCTS',
                                  help=_(messages.DISABLE_OPT_PRODUCTS_HELP),
                                  required=False)
+        self.parser.add_argument('--enabled-extended-product-search',
+                                 dest='enabled_extended_product_search',
+                                 nargs='+',
+                                 choices=scan.OPTIONAL_PRODUCTS,
+                                 metavar='ENABLED_EXTENDED_PRODUCT_SEARCH',
+                                 help=_(messages.SCAN_ENABLED_PRODUCT_HELP),
+                                 required=False)
+        self.parser.add_argument('--ext-product-search-dirs',
+                                 dest='ext_product_search_dirs',
+                                 nargs='+',
+                                 metavar='EXT_PRODUCT_SEARCH_DIRS',
+                                 help=_(messages.SCAN_EXT_SEARCH_DIRS_HELP),
+                                 required=False)
         self.source_ids = []
 
     def _validate_args(self):
         """Validate the edit arguments."""
         CliCommand._validate_args(self)
         # Check to see if args were provided
         if not(self.args.sources or self.args.max_concurrency or
-               self.args.disabled_optional_products):
+               self.args.disabled_optional_products or
```

Comment:
If you move to `*` you may have to differentiate whether the flag was supplied vs, whether you got back an empty list.